package.json

```diff
@@ -82,6 +82,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.13.10",
+    "@commitlint/lint": "^12.1.1",
     "@commitlint/load": "^12.0.1",
```

Comment:
Can you align `@commitlint/lint` and `@commitlint/load` versions for consistency's sake?

Reply:
done - I went with the newer version - all seems to still work as before